symplyphysics/laws/thermodynamics/enthalpy_change_via_entropy_change_and_pressure_change.py

```diff
@@ -0,0 +1,71 @@
+from sympy import Eq
+from symplyphysics import (
+    units,
+    Quantity,
+    Symbol,
+    Function,
+    print_expression,
+    validate_input,
+    validate_output,
+    symbols,
+)
+
+# Description
+## The fundamental thermodynamic relation are fundamental equations which demonstate how important
+## thermodynamic quantities depend on variables that are measurable experimentally.
+
+# Law: dH = T * dS + V * dp
+## H - [enthalpy](./enthalpy_is_internal_energy_plus_pressure_energy.py)
+## T - absolute temperature
+## S - entropy
+## p - pressure
+## V - volume
+## Notation: `d(x)` is an exact differential of `x`
+
+# Note
+## - Entropy `S` and pressure `p` are so called natural variables of enthalpy as a
+##   thermodynamic potential
+
+# Conditions
+## - The system is closed and homogeneous
+## - The system is in thermal equilibrium with the environment
+## - Only reversible prossesses or pure heat transfer are considered
+
+enthalpy_change = Function("enthalpy_change", units.energy)
```

Comment:
BTW why is this a function? I do not see it should be a function in original description. And what's point making it a function here?

Reply:
Because this differential sort of depends on the value of the other two differentials, at least that's how I imagine it in the absence of exact differentials. Ideally we should only have an enthalpy function which depends on entropy and pressure. I can also remove them and we will be left with symbols

Reply:
I think we should not have functions unless we differentiate or integrate them in the law.

Reply:
I see